promql/testdata/legacy.test

```diff
@@ -193,7 +193,7 @@ eval instant at 50m sum(http_requests) by (job) + min(http_requests) by (job) +
 	{job="app-server"} 4550 
 	{job="api-server"} 1750 
 
-
+# http_requests{job="app-server", instance="1", group="canary"}		0+80x10
```

Comment:
It was a line to help me verify/understand the test was acting as expected. removed.